### DISCUSSION TITLE
ERROR - Failed to load the extension "coqui_tts" #6075

### DISCUSSION THREAD

**Discussion by israelsouzaart** [oobabooga/text-generation-webui#6464] [General]
Já tentei de todas as formas e não consigo instalar as tais extensões, no final tudo acaba e erro e em frustrações, queria saber se há um profissional para instalar pra mim e qual valor para esse serviço.

**Comment by Palucas33** [comment]
eu mesmo problema

**Reply by italogbr-blip** [reply]
Ainda precisando? se estiver entra em contato via whatsapp 22997859221.

**Comment by italogbr-blip** [comment]
> Já tentei de todas as formas e não consigo instalar as tais extensões, no final tudo acaba e erro e em frustrações, queria saber se há um profissional para instalar pra mim e qual valor para esse serviço.

Ainda precisando? se estiver entra em contato via whatsapp 22997859221.

**Comment by kalle07** [comment]
maybe talk english ... so more can help you ^^

coqui_tts and all audio dont work at the moment, lot of requirements has changed
